app/controllers/physical_switch_controller.rb

```diff
@@ -0,0 +1,48 @@
+class PhysicalSwitchController < ApplicationController
+  include Mixins::GenericListMixin
+  include Mixins::GenericShowMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::MoreShowActions
+
+  before_action :check_privileges
+  before_action :session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  def self.table_name
+    @table_name ||= "switches"
+  end
+
+  def session_data
+    @title      = _("Physical Switches")
+    @layout     = "physical_switch"
+    @lastaction = session[:physical_switch_lastaction]
+  end
+
+  def set_session_data
+    session[:layout] = @layout
+    session[:physical_switch_lastaction] = @lastaction
+  end
+
+  def show_list
+    # Disable the cache to prevent a caching problem that occurs when
+    # pressing the browser's back arrow button to return to the show_list
+    # page while on the Switches' show page. Disabling the cache
+    # causes the page and its session variables to actually be reloaded.
+    disable_client_cache
```

Comment:
We don't support the browser's native back arrow button, so I don't think this is necessary.

Reply:
Done. I will update the repository soon.